docs/readme.md

```diff
@@ -35,26 +35,11 @@ request. Follow these steps to get the documentation site running locally.
 
 Forem's documentation is built with [GitDocs NodeJS library][gitdocs].
 
-The first step to running the documentations locally is to install the `GitDocs`
-package globally.
-
-With npm:
-
-```shell
-npm install gitdocs -g
-```
-
-Alternatively, you can use Yarn:
-
-```shell
-yarn global add gitdocs
-```
-
-Once installed, you should run `gitdocs serve` from the root of the `forem`
-project or from the `/docs` directory.
+To start the gitdocs server, you should run `yarn run gitdocs serve` from the
```

Comment:
It is not necessary to add the run command with yarn.

```suggestion
To start the gitdocs server, you should run `yarn gitdocs serve` from the
```

Reply:
I know but I wanted to make this as explicit as possible for people that don't know yarn and are used to npm.

Reply:
@abenettt good point, in the name of consistency I think we should apply the change though. I checked and all other commands in the documentation don't have `run` stated explicitly.

Reply:
Ah ok, I see that, I'll apply the suggestions then.